lang/test/org/partiql/lang/eval/EvaluatingCompilerCustomAnyOfTypeOperationTests.kt

```diff
@@ -42,10 +42,10 @@ class EvaluatingCompilerCustomAnyOfTypeOperationTests : CastTestBase() {
         private val esAnyCastIdentityCases = listOf(
             listOf(
                 // null/missing
-                case("NULL", "null", CastQuality.LOSSLESS) {
+                case("NULL", "NULL", CastQuality.LOSSLESS) {
                     assertEquals(ExprValueType.NULL, it.type)
                 },
-                case("MISSING", "null", FixSemantics(CastQuality.LOSSY)) {
+                case("MISSING", "MISSING", FixSemantics(CastQuality.LOSSY)) {
```

Comment:
It should be the PartiQL `MISSING`. The reason it was `null` there, simply because we were using Ion without annotation to represent it. 

With annotation, the Ion representation is `$missing::null`. And one test format was https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/test/org/partiql/lang/eval/evaluatortestframework/ExpectedResultFormat.kt#L25, which strip the annotation